tools/tpm2_listpcrs.c

```diff
@@ -271,14 +268,16 @@ static bool show_selected_pcr_values(listpcr_context *context) {
 
 static bool show_all_pcr_values(listpcr_context *context) {
 
-    init_pcr_selection_all(&context->algs, &context->pcr_selections);
+    if (!init_pcr_selection(0, context))
```

Comment:
normally we don't like to bury routines in conditionals, but ill let this slide.